zerver/migrations/0376_set_realmemoji_author_and_reupload_realmemoji.py

```diff
@@ -31,6 +33,14 @@ def set_emoji_author(apps: StateApps, schema_editor: DatabaseSchemaEditor) -> No
     RealmEmoji.objects.bulk_update(realm_emoji_to_update, ["author_id"])
 
     for realm_id in Realm.objects.order_by("id").values_list("id", flat=True):
+        if settings.TEST_SUITE:
+            print(
+                "\nThis migration is not meant to be applied to a provisioned test database.\n"
+                + "Canceling the problematic step to avoid errors, but it's recommended\n"
+                + "to run rebuild-test-database to ensure RealmEmoji are setup correctly.\n"
+            )
+            return
```

Comment:
I think we can just return without printing anything; our test suite data set has no custom emoji.